nonebot/adapters/discord/api/model.py

```diff
@@ -2298,6 +2298,19 @@ class VoiceRegion(BaseModel):
 
 # Webhook
 # see https://discord.com/developers/docs/resources/webhook
+class SourceGuild(BaseModel):
+    # partial guild object
+    id: Snowflake
+    name: str
+    icon: Optional[str]
```

Comment:
如果 `icon` 是 `Not required` 的
请在后面添加 `= None` 以适配 `Pydantic V2` 的行为
```suggestion
    icon: Optional[str] = None
```

Reply:
https://docs.pydantic.dev/2.7/migration/#required-optional-and-nullable-fields